packages/react-drag-drop/src/next/components/DragDrop/DragDropSort.tsx

```diff
@@ -57,6 +26,8 @@ export interface DragDropSortProps extends DndContextProps {
    * TableComposable variant wraps the draggable objects in TODO
```

Comment:
Theres is a lingering TODO in this prop description. I wonder if it could be cleaned up.